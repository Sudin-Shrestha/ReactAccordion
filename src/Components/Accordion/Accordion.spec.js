import accordion from './Accordions';

test('Accordion is working', () => {
    expect(accordion).toMatchSnapshot();
});