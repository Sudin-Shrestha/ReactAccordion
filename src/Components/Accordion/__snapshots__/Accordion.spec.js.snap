// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion is working 1`] = `[Function]`;
